js/gallerybutton.js

```diff
@@ -61,9 +61,7 @@ $(document).ready(function () {
 			GalleryButton.button =
 				$('<div id="gallery-button" class="button view-switcher">' +
 						'<div id="button-loading"></div>' +
-					'<img class="svg" src="' + OC.imagePath('core', 'actions/toggle-pictures.svg') +
-					'"' +
-					'alt="' + t('gallery', 'Gallery view') + '"/>' +
+					'<span class="icon-toggle-pictures"></span>' +
```

Comment:
Alt is in case of the image isn't loaded indeed. With title we should be able to have a popup. That's how we do on the other apps on nextcloud I think. There isn't any icons used as direct image (not that I'm aware of at least) :)